seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-3/src/test/resources/jdbc_snowflake_source_and_sink.conf

```diff
@@ -0,0 +1,66 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+env {
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
+
+source{
+    jdbc {
+        url = "jdbc:snowflake://<account_name>.snowflakecomputing.com"
+        driver = "net.snowflake.client.jdbc.SnowflakeDriver"
+        user = "user"
+        password = "password"
+        query = """
+        select C_ACCTBAL,
```

Comment:
Can you test all supported field types?
A rigorous test can refer to the following information:
The input and output of the test should include all supported data types.
2. The input data should be deterministic and the values of each row and column can be verified in the output.
3. In the output table, the values of each column in each row of data should be verified to ensure that there are no type conversion errors (precision loss, etc.) or missing values (a non null value in the input becomes null in the output) during the synchronization process.

Reply:
updated and the result is attached at https://github.com/apache/incubator-seatunnel/pull/4470#issuecomment-1542820279